src/monorepo-npm-publisher/src/transformFileVariants.js

```diff
@@ -11,9 +12,9 @@ export default function transformFileVariants(
   transpileESM: boolean,
 ): void {
   const getBabelConfig = (target: 'js' | 'js-esm' | 'flow') => {
-    // TODO: load Babel config from the packages instead (if exists)
     return {
-      root: __dirname, // do not lookup any other Babel config
+      root: findMonorepoRoot(),
+      rootMode: 'upward',
```

Comment:
if you set root as monorepo root, isn't `rootMode=upward` same as `rootMode=root`?

Reply:
Yes, I think you are right, I will remove the `root`